bootstrap/kubeadm/main.go

```diff
@@ -151,11 +152,13 @@ func main() {
 
 	// klog.Background will automatically use the right logger.
 	ctrl.SetLogger(klog.Background())
-
 	if profilerAddress != "" {
-		klog.Infof("Profiler listening for requests at %s", profilerAddress)
+		setupLog.Info("Profiler listening for requests at %s", profilerAddress)
```

Comment:
setupLog.Info has this signature:
```go
func (l Logger) Info(msg string, keysAndValues ...interface{}) {
```

Afaik this log line would panic because of uneven keysAndValues count

Reply:
Thanks for catching that! Replaced now.